main.py

```diff
@@ -73,9 +42,15 @@ def convert_website_to_stamp(_website, maximum_pages):
 
     url, maximum_pages = get_user_input()
 
-    _website = Website(url)
-
-    if not _website.is_valid:
-        LOGGER.error("Invalid URL!")
-    else:
-        convert_website_to_stamp(_website, maximum_pages)
+    _stampifier = Stampifier(url, maximum_pages)
+
+    try:
+        _stamp_file_path = _stampifier.stampify()
+    except InvalidUrlError as err:
+        LOGGER.debug(err.message, err.provided_url)
```

Comment:
include provided_url in message, so that only LOGGER.debug(err.message) would be sufficient.

Reply:
Updated. Thanks for the suggestion!